Sources/OpenFeature/EventHandler.swift

```diff
@@ -1,11 +1,20 @@
 import Foundation
+import Combine
 
 public class EventHandler: EventEmitter, EventPublisher {
     private let providerNotificationCentre = NotificationCenter()
 
     public init() {
     }
 
+    public func observe() -> Publishers.MergeMany<NotificationCenter.Publisher> {
+        // TODO Use sealed enum to ensure completeness
```

Comment:
if we move out of the `addHandler` paradigm, we can use [subjects](https://developer.apple.com/documentation/combine/currentvaluesubject) instead right? this subject will also fixes the problem of the ordering of observing .. meaning that if we start to observe later we replay the current value.